### ISSUE TITLE
What needs to be fixed 2022 + Images + Testing Files (All solved in 3.6)

### ISSUE THREAD

**Issue by DCGYT** [akai-katto/dandere2x#236]
### **Hi!**
I have already remastered my favorite series having to use two versions and mix the videos to decrease the errors, I would like these errors to be fixed in the next version of Dandere2x so I don't have to upscale twice and save time.

### **1. The corner points of the grid.**
Sometimes there are dots that stay on the grid, this happens when there is some white color on the screen and when it is gone, the corners keep that white color. I do not know if it is related to the error of the grid with white lines.

![image](https://user-images.githubusercontent.com/98280315/150692624-07c9af9e-7718-4019-b140-cd00dce0fa43.png)

### **2. The contrast**
I would like to keep the contrast as in previous versions, apparently the contrast is 116.27% higher than previous versions.
By lowering the contrast to 86% in this version, it remains the same as previous versions.

![image](https://user-images.githubusercontent.com/98280315/150692766-bd661aa9-09c8-464b-a2e6-c69a7a30e3cf.png)

Everything else I find excellent, because the performance has improved in the last versions. In addition to having fixed several bugs from previous versions such as dark squares in fade transitions, colors with a lot of noise and squares that did not change.

Testing Files: https://drive.google.com/drive/folders/1xXWt6FYannXfiZwwSfa4I3UCa9s0cmo4?usp=sharing

**Comment by eoiz** [comment]
What's you settings?
I use quality 100% and 60 for block size (almost never happened visual glitches, and also have the best compression for file size in my testing).

**Comment by DCGYT** [comment]
@eoiz
Windows 10 20H2
CPU: Intel Core i5 8300H
GPU: GTX 1050 Mobile 4GB
RAM: 12GB
Waifu2x Type: Waifu2x-Vulkan
Process Type: singleprocess
Denoise Level: Level 3
Scale Factor: 2 x
Image Quality 100
Block Size 60
You can use the test clips to experiment, the line test takes no more than 1 minute.  Play the result in full screen to see the faults more easily.

**Comment by eoiz** [comment]
> 
I use: i9 9900k + rtx 3080ti with 32gb of ram, after upscaling numerous series I haven't noticed the bug you found.
I do got some "square" glitches when I was using XMP ram, but after disabling it all videos upscaled just fine...
Also I'm using dandere2x 3.5 and 3.3...

**Comment by eoiz** [comment]
Sorry, i forgot to mention I have modiefied the output_options.yaml like this, don't know if it matters anyway:

> dandere2x:
  bleed: 3
> 
> dandere2x_cpp:
  block_matching_arg: "exhaustive"
  evaluator_arg: "mse"
> 
> ffmpeg:
  pre_process_video:
    -hwaccel: auto
    output_options:
      -pix_fmt: rgba64le
      -vsync: "cfr"
      -crf: 1
      -vf:
        - " null"
> 
 >  convert_video_to_gif:
    output_options:
      -pix_fmt: "yuv444p"
      -r: 15
> 
>  pipe_video:
    -hwaccel: auto
    output_options:
      -loglevel: panic
      -y: -f
      image2pipe: True
      -i: '-'
      -vcodec: libx265
      -preset: medium
      -qscale: 5
      -crf: 15
      -vf:
        - ' deband=range=8:blur=false, pp7=qp=2:mode=medium'
> 
>  migrate_audio:
    output_options:
      -c: copy
      -y: True

**Comment by DCGYT** [comment]
@eoiz Apparently the lines bug was fixed in Dandere2x version 3.5. But the corner points bug is still there. I don't know if it will have a solution.
I did the test with the default configuration and the one you have sent, but it still continues.
I think the error is because Dandere2x doesn't consider edges when comparing frames, so if a block ends with a blank edge, in the following frames it remains there.
This error disappears using smaller blocks.
